tests/features/common/kie-common.feature

```diff
@@ -124,4 +124,30 @@ Feature: RHPAM and RHDM common tests
       | AUTH_LDAP_LOGIN_MODULE | required  |
     Then file /opt/eap/standalone/configuration/standalone-openshift.xml should contain <login-module code="LdapExtended" flag="required">
 
+  Scenario: Check KIE_SERVER_JBPM_CLUSTER flag enabled
+    When container is started with env
+      | variable                        | value                |
+      | JGROUPS_PING_PROTOCOL           | kubernetes.KUBE_PING |
+      | KIE_SERVER_JBPM_CLUSTER         | true                 |
```

Comment:
Can you please also add test, where is set up jbm cluster timeout limit (e.g. to double value -> 120000) and chek the value

Reply:
Added